system/datatypes/peer_reviewed_article.php

```diff
@@ -1233,14 +1363,53 @@ private function sendNewCommentMail(HyphaDomElement $comment) {
 			'name' => $name,
 			'title' => $title
 		]);
-		$message = __('art-comment-body', [
+		$messageArgs = [
 			'name' => htmlspecialchars($name),
 			'link' => htmlspecialchars($linkToPage),
 			'title' => htmlspecialchars($title),
 			'comment' => nl2br(htmlspecialchars($commentBody)),
+		];
+		$message = __('art-comment-body', $messageArgs);
+		$this->sendMail(getUserEmailList(), $subject, $message);
+
+		$subscriberSubject = __('art-comment-subscriber-subject', [
+			'name' => $name,
+			'title' => $title
 		]);
+		$subscriberMessage = __('art-comment-subscriber-body', $messageArgs);
+
+		// also send newly create comment to all who subscribed to this discussion, with unsubscribe link
+		foreach ($this->getSubscribers($comment) as $subscriber) {
+			$email = $subscriber->getAttribute(self::FIELD_NAME_DISCUSSION_SUBSCRIBER_EMAIL);
+			$code = $subscriber->getAttribute(self::FIELD_NAME_DISCUSSION_SUBSCRIBER_UNSUBSCRIBE_CODE);
+			// include email for server record purposes
+			$path = hypha_substitute(self::PATH_UNSUBSCRIBE_CODE, ['address' => htmlspecialchars($email), 'code' => htmlspecialchars($code)]);
+			$linkToUnsubscribe = $this->constructFullPath($this->pagename . '/' . $path);
+			$message = $subscriberMessage;
+			$message .= '<p><a href="' . $linkToUnsubscribe . '">' . htmlspecialchars(__('art-unsubscribe')) . '</a></p>';
```

Comment:
I wonder if this unsubsribe stuff should be appended, or maybe included in the `__('art-comment-subscriber-body')` string?

Reply:
Don't want to risk it being excluded.